tsl/test/isolation/expected/compression_ddl.out

```diff
@@ -132,7 +132,7 @@ step A2: COMMIT;
 step C1: <... completed>
 compress_chunk 
 
-_timescaledb_internal._hyper_7_13_chunk
+_timescaledb_internal._hyper_7_12_chunk
```

Comment:
why did chunk numbering in the isolation tests change?

Reply:
I believe this is a result of changing the locking the source chunk in compress_chunk_impl from a AccessShareLock to a ShareLock.  This causes it to block before it creates the chunk to compress into in the previous permutation, where the compress ends up being rolled back due to the conflicting `ALTER`.